### DISCUSSION TITLE
How to copy entries from one specific site to other sites in different groups?

### DISCUSSION THREAD

**Discussion by whoiscado** [craftcms/cms#9979] [Q&A] [answered]
Hey, 

I have the problem that I can't find a "perfect" solution for my problem.

**actual state:**
- one website group with two different websites in two different languages

**target state:**
- many website groups that have one or two websites in each group
- the website groups differ in other languages

**Notes:**
- there's a high possibility that other languages (so other website groups and websites) will follow in the future
- all websites should have access to all sections

**What's the problem?** 
- one website (of the two existing ones) has "basic" entries that needs to be in all new websites too
  - all new created websites should have the same entries as this one
     - but after duplicating the entries to the new sites, they shouldn't have any kind of association and the default propagation method for **all** sections should be `save entries in the same site-group `

I know that I theoretically can save all entries across each sites white the propagation method where entries are saved to all active websites and change the method to `site-group only` after the duplication process queue finished. 
This doesn't work for my use case, as I'd need to deactivate one of my main sites for that and after enabling it, I'd lose my entries there, because it will copy the entries from the other site of the group.

There has to be a way to copy all entries (without association to other websites after duplicating) from one specific site to new ones?

Thanks **a lot** for any kind of help!!!

**Comment by brandonkelly** [comment, marked as answer]
Currently your best bet would be to export the entries as JSON (using the “Expanded” export type), and then import the data back in using [Feed Me](https://plugins.craftcms.com/feed-me).
